doc/usecases/plugins/spec_globbing.md

```diff
@@ -0,0 +1,47 @@
+# Use Case: Specification for REST application communicating with multiple services
+
+## Summary
+
+- **Title:** Specification for REST application communicating with multiple services
+- **Scope:** Configuration
+- **Level:** Developer Goal
+- **Actors:** User (usually a application developer)
+- **Brief:** User configures specifcation for configuration of REST application.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:** Specification is written and stored system wide or user specifc. All services which are added in the configuration need to adhere to this configuration specification.
+- **Alternative scenario:** User is limited to only using one service if globbing is not used. 
```

Comment:
This is not an alternative scenario but an alternative to the use case, which does not need to be written.

```suggestion
- **Alternative scenario:** None.
```